pom.xml

```diff
@@ -21,7 +21,7 @@
   <name>HTTP Plugins</name>
   <groupId>io.cdap</groupId>
   <artifactId>http-plugins</artifactId>
-  <version>1.4.0-SNAPSHOT</version>
+  <version>1.4.1-service-account</version>
```

Comment:
Please revert the version change heree.  We'll bump the version on the release branch after cherrypicking your PR.